packages/dds/tree/src/events/events.ts

```diff
@@ -95,7 +93,9 @@ export interface Listenable<TListeners extends object> {
 	 * @param eventName - the name of the event
 	 * @param listener - the handler to run when the event is fired by the emitter
 	 * @returns a {@link Off | function} which will deregister the listener when called.
-	 * This function has undefined behavior if called more than once.
+	 * This deregistration function is idempotent and therefore may be safely called more than once with no effect.
+	 * @remarks Avoid registering the exact same `listener` object for the same event more than once.
```

Comment:
given our current code throws (allowed under undefined behavior), I think stronger language is appropriate here

```suggestion
	 * @remarks Do not register the same `listener` object for the same event more than once.
```

Reply:
It's really strong now thx 💪